doc/aliases/add_youtube.html

```diff
@@ -0,0 +1,11 @@
+<script type="text/javascript">
+//<![CDATA[
+var $getDiv = $('div.addYoutube').last();
+var filmId = $getDiv.html();
+filmId = filmId.replace("https://","");
+filmId = filmId.replace("www.","");
+filmId = filmId.replace("youtube.com/watch?v=","");
+document.write("<div align='center'><iframe title='my title' width='560' height='349' src='http://www.youtube.com/embed/"+filmId+"?rel=0&loop=1' frameborder='0' allowfullscreen align='middle'> </iframe> </div>");
```

Comment:
this line is too long, split it in several ones (let's say of 120 characters most).